.stylelintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true
+  },
+  "ignoreFiles": [
+    "**/reset*.css",
+    "**/bootstrap*.css"
+  ]
+}
```

Comment:
- [ ] The `stylelintrc` file is different from the one provided by Microverse. We need to use [this](https://github.com/microverseinc/linters-config/blob/master/javascript/.stylelintrc.json) version instead.